Corona-Warn-App/src/main/java/de/rki/coronawarnapp/ui/eventregistration/organizer/create/TraceLocationCreateViewModel.kt

```diff
@@ -114,6 +158,24 @@ class TraceLocationCreateViewModel @AssistedInject constructor(
         }
     }
 
+    private class UpdateDelegateWithDefaultValue<T>(defaultValue: T) :
```

Comment:
The `UpdateDelegate` patterns do not seem easy to read or debug, but it's not a blocker for me here. Maybe we can come up with an even better solution to then also use in other screens :thinking: